src/server.js

```diff
@@ -365,6 +365,22 @@ function startBuilds(buildData) {
 	}));
 }
 
+/**
+ * Checks skip triggers and starts all builds needed for `prId` (which should be in the form 'User/Repo/123') as needed.
+ */
+function buildPullRequestIfNeeded(prId) {
+	const [owner, repo, prNumber] = prId.split('/');
+	return github.repos(owner, repo).pulls(prNumber).fetch()
```

Comment:
Should this Promise be returned? I can't see anything using it. (Or should something "listen" to it?)